src/axom/primal/geometry/OrientedBoundingBox.hpp

```diff
@@ -518,19 +518,15 @@ void OrientedBoundingBox<T, NDIMS>::addPoint(Point<T, NDIMS> pt)
 template <typename T, int NDIMS>
 void OrientedBoundingBox<T, NDIMS>::addBox(OrientedBoundingBox<T, NDIMS> obb)
 {
-  SLIC_CHECK_MSG(obb.isValid(), "Passed in OBB is invalid.");
```

Comment:
The description of the `addBox` function should be changed if it no longer throws an error with an invalid input. The current note says:
`* \note If obb is invalid, makes this invalid too.`